analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -287,7 +287,9 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
             aetherRepositorySystem
         )
 
-        return DefaultRepositorySystemSession(session).setWorkspaceReader(workspaceReader).apply {
+        val skipDownloadWorkspaceReader = SkipBinaryDownloadsWorkspaceReader(workspaceReader)
+
+        return DefaultRepositorySystemSession(session).setWorkspaceReader(skipDownloadWorkspaceReader).apply {
```

Comment:
Nit: I know it was like that before, but we could take the chance and move `setWorkspaceReader(skipDownloadWorkspaceReader)` to inside `apply`.